packages/odyssey-react/src/components/Icon/_icon-config.js

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+module.exports = {
+  "ignoreExisting": true,
+  "typescript": true,
+  "prettierConfig": {
+    "parser": "typescript",
+  },
+  "template": require('./_icon-template.js'),
+  "indexTemplate": require('./_index-template.js'),
+}
```

Comment:
it would be nice to move this config, the SVGR dependency and all the template files into a new private package: `@okta/odyssey-svgr` so that we can keep that complexity grouped together a bit and then consume that new private package from `@okta/odyssey-react`.

Also you can use typescript for this scripting wi the node12 tsconfig I setup which is helpful!